frontend/src/metabase/entities/containers/EntityCopyModal.tsx

```diff
@@ -6,6 +6,15 @@ import { t } from "ttag";
 import EntityForm from "metabase/entities/containers/EntityForm";
 import ModalContent from "metabase/components/ModalContent";
 
+interface EntityCopyModalProps {
+  entityType: any;
+  entityObject: any;
```

Comment:
<T>?

Reply:
maybe, but I'm not sure as I didn't find any example of similar type I believe

Reply:
I left it as any

Reply:
Can we maybe use `unknown` instead? `any` could be too loose

Reply:
I believe unknown is not the best type, as we'll need to cast it somehwhere anyway. What actually types can be used? I can think of Question,Dashboard,Collection?

Reply:
We can give it a shot unless it turns into a mess; hopefully we'll be able to provide more type-safety with entity hooks.

> What actually types can be used? I can think of Question,Dashboard,Collection?

I think there should be a type in `metabase-types/api` per each possible entity. Think we should use `Card` instead of `Question` tho

Reply:
I'm leaving it as it is for now as it should be generic most likely, but I met also different problem that Question from metabase-lib relies on metabase-types which is deprecated
https://github.com/metabase/metabase/blob/master/frontend/src/metabase-lib/Question.ts#L38

and it leads ot type error even using generics becuase of differences between Card from API and Card from metabase-types/types, which leads to further TS errors.

Probably likely it worth having an issue for this refactoring

Reply:
https://github.com/metabase/metabase/issues/30471